eng/common/SetupNugetSources.ps1

```diff
@@ -146,16 +146,22 @@ $userName = "dn-bot"
 # Insert credential nodes for Maestro's private feeds
 InsertMaestroPrivateFeedCredentials -Sources $sources -Creds $creds -Username $userName -Password $Password
 
-$dotnet3Source = $sources.SelectSingleNode("add[@key='dotnet3']")
```

Comment:
should we be modifying these?  since this is the arcade owned bit

Reply:
I am pulling in an updated Arcade https://github.com/dotnet/roslyn/pull/61044

Reply:
Ah - :shipit:

Reply:
Changed to build on latest NET3 arcade https://github.com/dotnet/roslyn/pull/61045